assets/hostfile

```diff
@@ -0,0 +1,5 @@
+10.88.56.135 compute-x86-01
```

Comment:
These values seem project specific.

Reply:
Perhaps this stub should be empty, with a comment block about how it will be used?

https://github.com/packet-labs/OpenStackOnPacket/pull/44/files#diff-78905f763f11dbcb0d3ea864d68b7937b68940df49b6531ef9e64eac76b3c5f5R7

Modifying the file within a module path is not safe for use with >1 instance of the module (which could be included from relative paths, be iterated over, or be invoked from the same pwd with a different `terraform workspace`).

Reply:
I'd suffice to handle those concerns with an open issue to revisit them.